viite-UI/src/model/SelectedProjectLink.js

```diff
@@ -21,9 +21,13 @@
     };
 
     var openWithErrorMessage = function (id, errorMessage) {
-      current = projectLinkCollection.getProjectLink([id]);
-      ids = [id];
-      eventbus.trigger('projectLink:errorClicked', get(id), errorMessage);
+      current = projectLinkCollection.getProjectLink(id);
+      ids = id;
+        if (!_.isUndefined(current[0].getData().connectedLinkId) && current.length > 2) {
+            openSplit(ids[0], current.length > 1);
```

Comment:
Since we are already inside of an "if" condition that validates "current.length > 2", we don't need to make the validation to "current.length > 1", we can just send "true" to OpenSplit method.